rs/embedders/tests/instrumentation.rs

```diff
@@ -1010,3 +1082,144 @@ fn table_modifications_are_unsupported() {
     let err = test("(table.init 0 (i32.const 0) (i32.const 0) (i32.const 0))");
     assert!(err.contains("unsupported instruction table.init"));
 }
+
+#[test]
+fn metering_wasm64_load_store_canister() {
```

Comment:
Perhaps another test where we check that instructions that cost the same, indeed result in the same output?

Reply:
makes sense, I'll do it like that.